TesseractCore.xcodeproj/project.pbxproj

```diff
@@ -54,8 +57,11 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		1182B03A1A84381B002E7D2D /* ImportingTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = ImportingTests.swift; path = TesseractCoreTests/ImportingTests.swift; sourceTree = SOURCE_ROOT; };
+		1182B03C1A843B63002E7D2D /* Madness.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Madness.framework; path = "../../Library/Developer/Xcode/DerivedData/TesseractCore-byzvyskleahrxbehbkbhivcpdwdh/Build/Products/Debug/Madness.framework"; sourceTree = "<group>"; };
```

Comment:
If you select `Madness.framework` in the file navigator and open the file utilities pane, there’s a little pop-up menu that allows you to select what the reference is relative to. Could you please set that to “Relative to Build Products”?

![screen shot 2015-02-07 at 11 08 26](https://cloud.githubusercontent.com/assets/59671/6092734/b08c6154-aeb9-11e4-90d8-4e1df0139308.png)